app/models/volunteer.rb

```diff
@@ -176,4 +183,13 @@ def to_s
   def record_acceptance_changed
     self["#{acceptance_change_to_be_saved[1]}_at".to_sym] = Time.zone.now if will_save_change_to_acceptance?
   end
+
+  def external_changed?
+    will_save_change_to_external?
```

Comment:
Hmm looks like the `#{attribute}_changed?` methods are [deprecated/renamed in Rails 5.1](https://github.com/rails/rails/pull/25337#issuecomment-225166796), but the [documentation isn't updated yet](http://api.rubyonrails.org/classes/ActiveModel/Dirty.html).

Did you add this to avoid the deprecation warning? In the long-term it's probably better to just use the new names, even though they're long and ugly :(

Reply:
If you want to keep it you can also use `alias_method` as a shortcut

Reply:
@chrysalag gave me the hint, I wasn't aware they're deprecated. But yea, the new naming isn't nice, I agree with that. But as this isn't a thing I'm going to look at all the time, I don't mind too much... 😉